resources/views/site/myMuseumTour.blade.php

```diff
@@ -112,7 +112,7 @@
                                         "restrict" => false,
                                     ];
                                 @endphp
-                                <a href="{{ config('aic.protocol') }}://{{ rtrim(config('app.url')) }}/artworks/{{ $artwork['id'] }}"
+                                <a href="{{ rtrim(config('app.url')) }}/artworks/{{ $artwork['id'] }}"
```

Comment:
I think this was meant to be `rtrim(config('app.url'), '/')`. Also, Laravel does provide a helper that provides this functionality: https://laravel.com/docs/11.x/helpers#method-url